vgen/src/main/java/io/gapi/vgen/DiscoveryFragmentGenerator.java

```diff
@@ -48,9 +48,10 @@ public static DiscoveryFragmentGenerator create(
 
     ApiaryConfig apiaryConfig = discovery.getConfig();
 
+    // TODO: Support multiple templates; don't hard-code the zero-index here.
```

Comment:
Maybe throw an exception if there is more than one, so that discovering the lack of support is easier when someone tries to use it down the road.